### PR TITLE
Feat: Added Questions Page

### PR DESCRIPTION
**Pull request by shuntzhere**
Added questions.html and questions.css
Screenshot (31)

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  --color-secondary: #aa2d59;
+  --bg-bright: #3cefff;
+  --bg-reef: #00f7ff;
+  --bg-dark: #4592af;
+
+  --color-white: #fdfdfd;
+  --color-black: #222222;
+  --color-shadow: #cfd9de;
+  /* font families */
+  /* font-family: "Lora", serif;
```
Comment (vivekmehto) [inline css/main.css:19]: remove unused code.

**Review by vivekmehto — APPROVED** [review]
looks good. one suggestion don't  add files that are already reviewed. add only question page files.

File: css/home.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+span {
+  color: var(--color-white);
+  letter-spacing: 0px;
+  font-size: 2rem;
+  transition: all 0.1s ease-in;
+}
+
+span:hover {
```
Comment (Sameony) [inline css/home.css:55]: avoid giving styling directly to the elements

**Review by Sameony — APPROVED** [review]
do follow up on these minor suggestions before merging

File: css/home.css (hunk 2 above)
Comment (shuntzhere) [reply css/home.css:55]: Sure, will be giving class names to avoid this in future.

**Pull request merged**